app/components/mandatarissen/timeline.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import moment from 'moment';
+
+export default class MandatarissenTimeline extends Component {
+  @service features;
+  @service store;
+
+  get isFeatureEnabled() {
+    return this.features.isEnabled('shacl-report');
```

Comment:
Is this the correct feature flag?
```suggestion
    return this.features.isEnabled('timeline');
```